core/src/View/Kokkos_ViewTracker.hpp

```diff
@@ -62,7 +62,7 @@ struct ViewTracker {
   }
 
   template <class RT, class... RP>
-  KOKKOS_INLINE_FUNCTION void assign(const View<RT, RP...>& vt) noexcept {
+  KOKKOS_INLINE_FUNCTION void assign(const View<RT, RP...>& vt) {
```

Comment:
Now we have other members that are marked noexcept and call `assign`...
I did not manage to see this warning locally.  What build had a warning here?

Reply:
`clang-tidy` complains with the `Threads` backend.